vyper/functions/functions.py

```diff
@@ -85,6 +93,14 @@ class Floor:
     _inputs = [("value", "decimal")]
     _return_type = "int128"
```

Comment:
Yeah, because the input is always a decimal. We could change this in the future tho...  50% of cases, `uint256` will be fine, just needs an underflow check.

Reply:
Well, I mean it should allow more than 1 type (basically, pass through input type)

But, I think this is talking about what we ultimately want, not what we have today.